markdown/blog/community-spotlight-ashmit.md

```diff
@@ -0,0 +1,136 @@
+
+**Bami: Did you face any unexpected challenges or surprises during the program?**
+
+***Ashmit:** Apart from the final submission issue, everything went smoothly. However, the most unexpected challenge was adapting to remote collaboration. Before GSoC, I wasn’t very experienced with remote work. The program helped me develop skills in asynchronous communication and project management, which are crucial in open source and professional environments alike.*
+
+## Key Takeaways and Advice for Future GSoC Mentees
+
+**Bami: What were your key takeaways from the GSoC experience?**
+
+***Ashmit:** One of my biggest takeaways was that anything is achievable with hard work and patience. Initially, I doubted my chances of being selected, but I put in the effort and succeeded. Another key lesson was the importance of structured workflows in open-source projects, which significantly improves maintainability.*
+
+**Bami: Has this experience changed your perception of open source? Was this your first time contributing?**
+
+***Ashmit:** No, this wasn’t my first open-source experience. I previously participated in Hacktoberfest 2021 when they still gave out T-shirts! However, being a GSoC mentee and later a maintainer was a completely different experience. As a mentee, you focus on individual contributions, whereas as a maintainer, you oversees multiple contributions and ensure the stability of the project. It gave me a deeper appreciation of the responsibilities maintainers handle.*
+
+**Bami: Any advice for future GSoC mentees or newcomers to AsyncAPI?**
+
+***Ashmit:** I have two main pieces of advice: be patient and work hard. Open source is community-driven, and maintainers often have full-time jobs, so responses to contributions can take time. Patience is essential. Hard work is equally important—keep learning, refining your skills, and making meaningful contributions.*
+
+## Looking Ahead: What’s Next for Ashmit?
+
+**Bami: What are your plans after GSoC? Will you continue contributing to AsyncAPI or explore other open-source projects?**
+
+***Ashmit:** Right now, I am focusing on completing my degree and securing a position in a good software company. However, I will continue contributing to open source because it has given me so much, and I feel a responsibility to give back. While AsyncAPI remains my primary focus, I am also interested in projects like Chromium, though I know it’s quite a challenging one to take on.*
```

Comment:
```suggestion
***Ashmit:** I am currently focusing on completing my degree and securing a position in a good software company. However, I will continue contributing to open source because it has given me so much, and I feel a responsibility to give back. While AsyncAPI remains my primary focus, I am also interested in projects like Chromium, though I know it’s quite a challenging one to take on.*
```